<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_normes</class>
 <widget class="QDialog" name="add_normes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Standard Addition</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255,255,255);</string>
  </property>
  <widget class="QLineEdit" name="le_type_normes">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Type..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_id_normes">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>ID..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_validity_normes">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Validit√©..</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="le_date_normes">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 9px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_sector_normes">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Secteur..</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ajouter_normes">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(158,59,59);
font: 75 9pt &quot;Arial&quot;;
border: 2px solid rgb(158,59,59);
border-radius: 15px;
}

QPushButton:hover {
  color: #fff; 
  background-color: #a63b3b; 
  border-color: #a63b3b; 
}

QPushButton:pressed {
  color: #fff; 
  background-color: #802c2c; 
  border-color: #802c2c; 
}</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/new/prefix1/gfx/3516835.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>le_type_normes</zorder>
  <zorder>le_id_normes</zorder>
  <zorder>le_validity_normes</zorder>
  <zorder>le_date_normes</zorder>
  <zorder>le_sector_normes</zorder>
  <zorder>b_ajouter_normes</zorder>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
